src/Common/Logging/EventAuditLogger.php

```diff
@@ -844,6 +844,55 @@ public function recordLogItem($success, $event, $user, $group, $comments, $patie
         $this->sendAtnaAuditMsg($user, $group, $event, $patientId, $success, $comments);
     }
 
+    /**
+     * Log HTTP request details
+     */
+    public function logHttpRequest()
+    {
+        // Skip if audit logging or http request logging is disabled
+        if (empty($GLOBALS['enable_auditlog']) || empty($GLOBALS['audit_events_http-request'])) {
+            return;
+        }
+
+        // Rest of the existing method code remains the same...
+        // Skip certain paths we don't want to log
+        // TODO: @adunsulag do we want to skip the log page? I think that's useful auditing information
+//        if (strpos($_SERVER['SCRIPT_NAME'], 'logview.php') !== false) {
+//            return; // Don't log requests to logging pages
+//        }
+
+        // Map HTTP methods to event action types
+        $methodMap = [
+            'GET' => 'select',
+            'POST' => 'update',
+            'PUT' => 'update',
+            'DELETE' => 'delete',
+            'PATCH' => 'update'
+        ];
+
+        $method = $_SERVER['REQUEST_METHOD'];
+        $event = $methodMap[$method] ?? 'select';
+
+        // Build the comment with path and query params
+        $comment = $_SERVER['SCRIPT_NAME'];
+        if (!empty($_SERVER['QUERY_STRING'])) {
+            $comment .= '?' . $_SERVER['QUERY_STRING'];
+        }
+
+        // Record the log entry
+        $this->newEvent(
+            "http-request-$event",  // event
+            $_SESSION['authUser'] ?? null, // user
+            $_SESSION['authProvider'] ?? null, // groupname
+            1, // success
+            $comment, // comments
+            $_SESSION['pid'] ?? null, // patient_id
+            'http-request', // log_from
+            null, // menu_item
+            0 // ccda_doc_id
```

Comment:
rec dropping the last 3 parameters since these are just used by the portal logging.

Reply:
Fixed